.github/workflows/main.yml

```diff
@@ -15,9 +15,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Store Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: ${{ runner.os }}-gradle-
+
       - name: Set up Android SDK
         uses: malinskiy/action-android/install-sdk@release/0.0.7
 
+      - name: Store Android SDK
```

Comment:
Could you add a comment that says that this won't work until https://github.com/Malinskiy/action-android/issues/26 is addressed?

Reply:
This half-work:
- This step cannot be put before the "Set up Android ADK" because of the issue => tools, platform-tools, licenses are downloaded
- By being there, this step remove all the build-tools setup done by gradle

I will add a comment explaining this.

Reply:
Done